Amplify/Categories/DataStore/Model/Internal/Schema/ModelSchema+Definition.swift

```diff
@@ -26,6 +26,14 @@ public enum ModelFieldType {
     case model(name: ModelName)
     case collection(of: ModelName)
 
+    public static func model(type: Model.Type) -> ModelFieldType {
+        return .model(name: type.modelName)
```

Comment:
nit: remove redundant `return`
```suggestion
        .model(name: type.modelName)
```